### DISCUSSION TITLE
Proprietary, NON-ATAPI CD-ROM drive emulation (SBCD, Mitsumi, etc)

### DISCUSSION THREAD

**Discussion by agithubuser444** [86Box/86Box#3050] [Ideas]
Hello, would it be possible to add Proprietary CD-ROM Emulation to 86Box, like for example, SBCD/Panasonic CD-ROM Drive support for the Sound Blaster Pro, or the Mitsumi CD-ROM Controller Card and CD Drive, these would be nice to emulation as they predate the ATAPI Standard, and would be cool to see in 86Box, mostly for historical purposes.

**Comment by Banjo-Oz** [comment]
Seconded! I actually discovered 86Box originally while looking to see if this could be done on PCem. :)
